api/Body.json

```
@@ -486,14 +486,12 @@
               "version_added": "44"
             },
             "safari": {
-              "version_added": "10.1",
-              "partial_implementation": true,
-              "notes": "See <a href='https://webkit.org/b/212858'>WebKit bug 212858</a>"
+              "version_added": false,
+              "notes": "This method was added in Safari 10.1, but always rejects with <code>NotSupportedError</code>. See <a href='https://webkit.org/b/215671'>bug 215671</a>."
```

Style-only suggestion here (and I need to do something about #6229)

```suggestion
              "notes": "From Safari 10.1, the method exists but always rejects with <code>NotSupportedError</code>. See <a href='https://webkit.org/b/215671'>bug 215671</a>."
```

---

I went about updating before spotting these suggestions. I've applied them and credited you manually in the commit message.